src/Storage/Commands.Storage.Test/Scripts/GetBillableSize.ps1

```diff
@@ -0,0 +1,283 @@
+﻿# this script will show how to get the total size of the blobs in a container
+# before running this, you need to create a storage account, create a container,
+#    and upload some blobs into the container
+# note: this retrieves all of the blobs in the container in one command.
+#       connect Azure with Login-AzureRmAccount before you run the script.
+#       requests sent as part of this tool will incur transactional costs. 
+# command line usage: script.ps1 -ResourceGroup {YourResourceGroupName} -StorageAccountName {YourAccountName} -ContainerName {YourContainerName}
+#
+
+param(
+    [Parameter(Mandatory=$true)]
+    [string]$ResourceGroup,
+
+    [Parameter(Mandatory=$true)]
+    [string]$StorageAccountName,
+
+    [Parameter(Mandatory=$true)]
+    [string]$ContainerName
+)
+
+#Set-StrictMode will cause Get-AzureStorageBlob returns result in different data types when there is only one blob
+#Set-StrictMode -Version 2
+
+$VerbosePreference = "Continue"
+
+if((Get-Module -ListAvailable Azure) -eq $null)
+{
+    throw "Azure Powershell not found! Please install from http://www.windowsazure.com/en-us/downloads/#cmd-line-tools"
```

Comment:
@EmmaZhu do you have any objection to using the following link instead?

https://docs.microsoft.com/en-us/powershell/azure/install-azurerm-ps

Reply:
Thanks for your comments. Changed and tested module installing and checking as the suggestions.

Reply:
To answer the above question"Out of curiosity, what is the intended usage of this script?", seems I cannot reply to it directly:
The script will be a sample for Azure Storage customers to reference to, to get billable size of an Azure Storage container.  We will have an article to introduce it.